servicetalk-loadbalancer/src/main/java/io/servicetalk/loadbalancer/RoundRobinLoadBalancer.java

```diff
@@ -618,14 +630,449 @@ public Completable closeAsyncGracefully() {
     }
 
     @Override
-    public List<Map.Entry<ResolvedAddress, List<C>>> usedAddresses() {
+    public List<Entry<ResolvedAddress, List<C>>> usedAddresses() {
         return usedHosts.stream().map(Host::asEntry).collect(toList());
     }
 
     private static boolean isClosedList(List<?> list) {
         return list.getClass().equals(ClosedList.class);
     }
 
+    private static final class Host<Addr, C extends LoadBalancedConnection> implements ListenableAsyncCloseable {
```

Comment:
This is copied back from commit 53ae43837fd31: since we're going to pull apart `Host` really soon, and that is a core part of how old RR works, I think we should just freeze that part as well.